tests/ubiquiti_edgerouter/show_interfaces/ubiquiti_edgerouter_show_interfaces.raw

```diff
@@ -0,0 +1,10 @@
+Codes: S - State, L - Link, u - Up, D - Down, A - Admin Down
+Interface    IP Address                        S/L  Description                 
+---------    ----------                        ---  -----------                 
+eth0         -                                 u/u  Port                        
+eth1         192.168.1.1/24                    u/u  datas                                
+             2a05:c100:1d::1/64
+eth2         -                                 u/D  Port                        
+eth3         -                                 u/D  Port
+lo           127.0.0.1/8                       u/u                              
+             ::1/128                          
```

Comment:
Add 2 dummy interfaces, one with only an IPv4 address, and one with only an IPv6 address.
You could use eth2 and eth3. In that way you would have all cases.

Also, make one Administratively down